src/models/ComponentTemporaryLink.js

```diff
@@ -0,0 +1,38 @@
+import ComponentLink from './ComponentLink';
+
+/**
+ * Class that describe a link from Components to nothing.
+ */
+class ComponentTemporaryLink extends ComponentLink {
+  /**
+   * Default constructor.
+   * @param {object} [props] - Object that contains all properties to set.
+   * @param {string} [props.anchorName] - Name of target where begin the link.
```

Comment:
Name of the target where the link begins.

Reply:
done